src/cache.rs

```diff
@@ -217,7 +217,7 @@ impl Cache {
 
         // Get cache dir
         let cache_dir = match Self::get_cache_dir() {
-            Ok((cache_dir, _)) => cache_dir.join("tldr-master"),
+            Ok((cache_dir, _)) => cache_dir.join("tldr-main"),
```

Comment:
This will result in a "cache directory missing" warning, and two caches side by side.

@niklasmohrin what do you think, should we simply keep the name "tldr-master" so that the cache is re-used? Should we rename the old directory if present?